src/main/java/carRace/step5/domain/CarRacingGame.java

```diff
@@ -1,99 +1,79 @@
 package carRace.step5.domain;
 
-import carRace.step5.domain.util.CarRasingDraw;
+import carRace.step5.domain.util.TryCount;
+import carRace.step5.view.CarRasingDraw;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Random;
+import java.util.*;
 
 public class CarRacingGame {
     private static Random random = new Random();
-    private HashMap<Object, List<Car>> carResultData = new HashMap<>();
-    private String[] carNames;
+    private List<Car> carsScore = new ArrayList<>();
     private static int raceTryCount;
 
-    public CarRacingGame(String[] carNames, int raceTryCount) {
-        this.carNames = carNames;
+    public CarRacingGame(int raceTryCount) {
         this.raceTryCount = raceTryCount;
     }
 
-    public void playRacingGame() {
-        List<Car> list = new ArrayList<>();
-
-        for (String carName : carNames) {
-            Car car = new Car(carName);
-            car.moveCar(random.nextInt(10));
-            list.add(car);
-
-            repeatRaceTryCount(list);
-            list = new ArrayList<>();
-        }
-    }
-
-    public void drawRasingWinner() {
-        List<Car> lastGameList = getLastPlay();
-        int maxValue = getMaxWinnerSocre(lastGameList);
-        List<Car> winnerList = new ArrayList<>();
-
-        for (int i = 0; i < lastGameList.size(); i++) {
-            selectWinner(maxValue, winnerList, lastGameList.get(i));
-        }
-
-        System.out.print("우승자 : " + winnerList.get(0).getCarName());
-        for (int i = 1; i < winnerList.size(); i++) {
-            System.out.print(", " + winnerList.get(i).getCarName());
-        }
+    public void RunPlayGame(String[] carNames) {
+        makeCars(carNames);
+        playRasingGame();
+        drawRasingWinner();
     }
 
-    public void drawRasingGame() {
+    private void playRasingGame() {
         for (int i = 0; i < raceTryCount; i++) {
-            for (String carName : carNames) {
-                System.out.print(carName + " : "
-                 + CarRasingDraw.changeNumberToChar(carResultData.get(carName).get(i).getMoveDistance()));
-                System.out.print(System.lineSeparator());
-            }
+            repeatRaceTryCount();
             System.out.print(System.lineSeparator());
         }
     }
 
-    private void repeatRaceTryCount(List<Car> paramList) {
-        Car car;
-        String carName = paramList.get(0).getCarName();
-
-        for (int i = 1; i < raceTryCount; i++) {
-            car = new Car(paramList.get(i - 1));
-            car.moveCar(random.nextInt(10));
-            paramList.add(car);
+    private void repeatRaceTryCount() {
+        for (int i = 0; i < carsScore.size(); i++) {
+            carsScore.get(i).moveCar(random.nextInt(10));
+            CarRasingDraw.printCarDistance(carsScore.get(i).getCarName(), carsScore.get(i).getMoveDistance());
```

Comment:
다음과 같이 줄여보면 어떨까요 :) ?
```suggestion
        for (Car car : carScore) {
            car.moveCar(random.nextInt(10));
            CarRasingDraw.printCarDistance(car.getCarName(), car.getMoveDistance());
```